integration-tests/features/environment.py

```diff
@@ -284,6 +284,21 @@ def _send_json_file(endpoint, filename):
     return response
 
 
+def _check_env_for_remote_tests(env_var_name):
+    if os.environ.get(env_var_name):
+        print("Note: {e} environment variable is specified, but tests are " \
+              "still run locally\n" \
+              "Check other values required to run tests against existing " \
+              "deployent".format(e=env_var_name))
+
+
+def _check_api_token_presence():
+    if not os.environ.get("RECOMMENDER_API_TOKEN"):
+        print("Warning: the RECOMMENDER_API_TOKEN environment variable is not" \
+              " set.\n"
+              "         Most tests that require authorization will probably fail")
```

Comment:
Maybe it would worth to tag these tests with something like "requires_api_token" tag - see https://pythonhosted.org/behave/tutorial.html#controlling-things-with-tags